internal/mysql/param_test.go

```diff
@@ -118,20 +128,20 @@ func TestInsertParamSearcher(t *testing.T) {
 
 	tests := []testCase{
 		testCase{
-			input: "INSERT INTO users (first_name, last_name) VALUES (?, ?)",
+			input: "INSERT INTO users (first_name, last_name) VALUES (?, sqlc.arg(UserLastName))",
```

Comment:
One thing I didn't mention in #71. I'm not sure we should allow mixing `?` and `sqlc.arg()`. Thoughts?